go.mod

```diff
@@ -97,9 +97,6 @@ require (
 )
 
 replace (
-	golang.org/x/crypto => golang.org/x/crypto v0.8.0 // CVE-2021-43565
```

Comment:
Do we not use /x/crypto anymore? In the go.sum I only see go.mods...

Reply:
It seems not for 2.8...